doc/team.rst

```diff
@@ -0,0 +1,91 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+We are an international team dedicated to building a Pythonic API for the
+Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
+new and experienced users by creating user-friendly interfaces with the GMT
+C API, supporting rich display in Jupyter notebooks, and integrating with
+the PyData ecosystem.
+
+Maintainers
+-----------
```

Comment:
'Maintainers' is correct, but I'd prefer a term like "Core team" or similar so that people won't feel like joining the team means maintaining software only (which is part of our job, but not the whole of it). Any other suggestions?

Reply:
**Core Developer**?

Reply:
Yes, I agree with your point. I think that specifying who leads project maintenance could be helpful transparency for new contributors, so it would be nice to get a solution that recognizes both maintenance, development, and the rest. Do you think it could be an option to have a 'Distinguished Contributors' section in addition to the 'Maintainers' section, with overlap between the two as appropriate? I like Jupyter's [distinguished contributors](https://jupyter.org/about) section because the path for recognition is clearly described. Or we could do a 'Core Developers' team like @core-man suggested, with some tag for people who are also maintainers. But, it is possible that someone may be interested in being a maintainer without wanting to spend as much time on development.

Reply:
Just want to follow up on this suggestion @weiji14 and @core-man, do you prefer a single "core team" to having two categories, one for active maintainers and one for core developers/core contributors/distinguished contributors? My preference is the second option, but I am open to either.

Reply:
> I think that specifying who leads project maintenance could be helpful transparency for new contributors, so it would be nice to get a solution that recognizes both maintenance, development, and the rest.

Sounds good. This could help new contributors to have a quick recognition of the PyGMT Team. But the current core developers are all maintainers, right?

> do you prefer a single "core team" to having two categories, one for active maintainers and one for core developers/core contributors/distinguished contributors?

If we only create one team called **core team**, do we have to create a Level-2 title for it (e.g., **Maintainers** and **Emeritus Maintainers** are Level-2 titles now)? Currently, the level-1 title is called **PyGMT Team Gallery**.

We have a few categories and they could be active or not active. I am a little confused about them.
- **Maintainers**
- **Core Developers**
- **Core Contributors**
- **Distinguished Contributors**
- **Contributors**

I think all of you belong to **Core Developers**, and all of you are also maintainers, right? So, I don't think we need to distinguish **Core Developers** and **Maintainers** unless we have one core developer who is not a maintainer, vice versa.

Could we add a labeler, **active**, for those core developers/maintainers who are active now? Thus, it does not need to create a separate category for core developers/maintainers who are not active now.

> My preference is the second option, but I am open to either.

:smile: Sorry that I'm not clear what the first and second options are.

Reply:
OK, I will make a revision accordingly and then you can check it out to let me know if I understood your comments correctly.

Reply:
I pushed a revised version based on my interpretation of your comments. I think we need a statement at the top that invites people to get involved and (ideally) lays out the criteria for becoming part of the 'core team' (if this iteration stays). To be honest, while I have spent enough time on PyGMT maintenance to have been comfortable with the maintainers group, it feels odd to me to be the one including myself and listing others on the 'core team' group when I have only been involved with PyGMT for a small amount of time compared to, for example, Wei Ji and Dongdong. I would rather it be better defined somewhere how the 'core team' gets decided and adjusted over time (maybe that guidance exists already, I am not sure).

Reply:
> To be honest, while I have spent enough time on PyGMT maintenance to have been comfortable with the maintainers group, it feels odd to me to be the one including myself and listing others on the 'core team' group when I have only been involved with PyGMT for a small amount of time

It is not the amount of time you've spent, but rather, what you can do that counts :)

At this stage, PyGMT is still a fairly small team (<10 people), and @core-man raises a good point that there's not much of a distinction between developers or maintainers per se. I'd be tempted to throw @core-man into the PyGMT team gallery list since he's been very helpful with a lot of the documentation and design/decision making.

> Could we add a labeler, active, for those core developers/maintainers who are active now? Thus, it does not need to create a separate category for core developers/maintainers who are not active now. We only have two non-active core developers now.

What about an asterisk (*) after the name to indicate that a person is active? Or have the name be **bold** when someone is active?

Reply:
> At this stage, PyGMT is still a fairly small team (<10 people), and @core-man raises a good point that there's not much of a distinction between developers or maintainers per se. I'd be tempted to throw @core-man into the PyGMT team gallery list since he's been very helpful with a lot of the documentation and design/decision making.
> 
Fully agree!

> > Could we add a labeler, active, for those core developers/maintainers who are active now? Thus, it does not need to create a separate category for core developers/maintainers who are not active now. We only have two non-active core developers now.
> 
> What about an asterisk (*) after the name to indicate that a person is active? Or have the name be **bold** when someone is active?

If possible, I would prefer somekind of label as suggested by @core-man.

Reply:
I like the current design, and I think any way above to indicate active maintainers/developers are okay. When we have a way to indicate active maintainers/developers soon, new contributors could easily recognize the whole team due to the limited number.

BTW, I feel new contributors could also know the details about PyGMT Teams by [pygmt-team](https://github.com/orgs/GenericMappingTools/teams/pygmt-team/teams), which has a very good and clear layout.

> I'd be tempted to throw @core-man into the PyGMT team gallery list since he's been very helpful with a lot of the documentation and design/decision making.

It feels odd to me to be included in the PyGMT team gallery list as compared to the big amount of work and time you spent. Meanwhile, I am afraid that I will still have limited time to help with only a few parts of PyGMT (i.e., tutorials/gallery/documentation review, and some module enhancement/wrapping interesting to me (e.g., `plot`/`plot3d`/`sac`/`wiggle`/`histogram` currently))

Reply:
Ok, I see that people are keen on having an **active** label/tag somehow. Will leave it up to @meghanrjones on how to include this in the team gallery and make it look nice :slightly_smiling_face:

> > I'd be tempted to throw @core-man into the PyGMT team gallery list since he's been very helpful with a lot of the documentation and design/decision making.

> It feels odd to me to be included in the PyGMT team gallery list as compared to the big amount of work and time you spent. Meanwhile, I am afraid that I will still have limited time to help with only a few parts of PyGMT (i.e., tutorials/gallery/documentation review, and some module enhancement/wrapping interesting to me (e.g., plot/plot3d/sac/wiggle/histogram currently))

Trust me @core-man, you've probably done more in the past three months in terms of features/documentation/review work than what I did in my first year with PyGMT :laughing:. And it's ok to focus on the parts you use such as the seismological modules. I would encourage you to be on the team list just to show that PyGMT values new contributors that can do good work and isn't restricted to an exclusive club of developers, but I respect your decision if you don't feel ready to be added yet.

Reply:
https://github.com/GenericMappingTools/pygmt/pull/1308/commits/9dbf864ef42873e83467ddbd47c516b7831f23fb added an 'active' label.

https://github.com/GenericMappingTools/pygmt/pull/1308/commits/52d814123ca516c1aac9094c13534e8972846237 added @core-man. @core-man, I will respect your preferences for whether you want to accept/reject inclusion on the page, so feel free to let me know. I agree with all the previous comments that your contributions are really valuable.

https://github.com/GenericMappingTools/pygmt/pull/1308/commits/d302e67baea0ac5f1d46fe995de932b9d38307dc restructured the page following Jupyter's Distinguished Contributors model, with a label of 'Maintainer' for active maintainers. I prefer this because it meets the two main goals of 1. acknowledging people who have put a lot into the project without requiring any responsibilities from them and 2. providing helpful transparency about who is responsible for the current maintenance of the project. I also prefer having it stated when/how it gets updated. Easy to revert if others do not like it.

Reply:
Thanks a lot for including me in the team gallery. I will continue to do what I can do for the project.